_posts/2023-01-11-install-and-manage-applications.md

```diff
@@ -0,0 +1,342 @@
+
+- If you have disabled Flatpaks, you can install applications as a Snap through GNOME Software.
+
+![GNOME Software](/assets/uploads/gnome-software-43.webp)
+
+### Searching Applications
+
+- You can search for applications in GNOME Software by clicking on the Search icon in the top left corner.
+- If the application you searched for exists, it will be displayed, and you can proceed with the installation.
+- If the application you searched for doesn't exist, you can use an alternative method suggested below in this guide to install it.
+
+### Updating Applications
+
+- Updates are visible in the `Updates` panel in GNOME Software. In Vanilla OS, Updates are available for Snaps and Flatpaks through it. 
+- Automatic Updates are configured by clicking on `Update Preferences` in the Sidebar and enabling Automatic Updates. Optionally, you can switch on `Automatic Update Notifications` to be informed about updated applications.
+
+![GNOME Software Update Proferences](/assets/uploads/gnome-software-update-preferences.webp)
+
+### Removing Applications
+
+- You can remove the installed Flatpaks and Snaps from the Installed Panel in GNOME Software. 
+(Note: Some native applications installed in the host will not support removing using this method in Vanilla OS)
+
+## Installing AppImages
+
+- AppImages is one of the recommended ways to execute applications. If you have enabled AppImages in the first setup, You can run it graphically using the following steps:-
+    - Right-click on the file, then click on `Properties`.
+    
+   ![AppImages Properties](/assets/uploads/appimages-nautilus-properties.webp)
+    
+    - Enable the `Executable as Program` option.
+    - Now, you can run the AppImage by right-clicking it and pressing run or by pressing enter/return key on the keyboard.
+
+Alternatively, you can open AppImages from the command line using the following commands:-
+
+```bash
+cd <directory>
+chmod +x <file>.appimage
+./<file>.appimage
+```
+
+Note:- `chmod +x <file>.AppImage` makes the file executable.
+
+ ![Running Krita using AppImage](/assets/uploads/appimages-nautilus-properties.webp)
+
+## Installing Flatpaks from the Command line
+
+- Flatpaks if enabled in the First Setup, can be installed from the command line using the command:-
+
+```bash
+flatpak install flathub <application-id>
+```
+- You can run the applications using the desktop entry (icon) from the application menu, or you can run it manually through:-
+
+```bash
+flatpak run <application-id>
+```
+- You can remove installed applications using the following command:-
+
+```bash
+flatpak uninstall <application-id>
+```
+
+- You can remove all the leftover data of the application using the following command (Disclaimer: Proceed with caution as this might lead to data loss):-
+
+```bash
+flatpak uninstall <application-id> --delete-data
+```
+
```

Comment:
Maybe it could be interesting to also document `flatpak uninstall --unused` to clean dependencies.

Reply:
Thanks, I will add it too tomorrow.